rust.007.rawio/src/main.rs

```diff
@@ -0,0 +1,157 @@
+#![feature(slice_internals)]
```

Comment:
Same counting strategy as rust.003, but it really shines here because we can pass the simd code big chunks of bytes at a time.